mlearning/libnnablart/nnablart.defs

```diff
@@ -0,0 +1,42 @@
+############################################################################
+# apps/mlearning/libnnablart/nnablart.defs
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.  The
+# ASF licenses this file to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance with the
+# License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+############################################################################
+
+ifeq ($(CONFIG_NNABLA_RT),y)
+
+nnabla.zip:
+	$(Q) curl -L https://github.com/sony/nnabla-c-runtime/archive/refs/tags/v$(CONFIG_NNABLA_RT_VER).zip -o nnabla.zip
+	$(Q) unzip -o nnabla.zip
+	$(Q) mv nnabla-c-runtime-$(CONFIG_NNABLA_RT_VER) nnabla-c-runtime
+
+.nnabla_headers: nnabla.zip
+	$(shell mkdir -p $(TOPDIR)$(DELIM)include$(DELIM)nnablart$(DELIM))
+	$(eval headers := $(wildcard nnabla-c-runtime$(DELIM)include$(DELIM)nnablart$(DELIM)*.h))
+	$(foreach header,$(headers),$(shell cp -rf $(header) $(TOPDIR)$(DELIM)include$(DELIM)nnablart))
```

Comment:
can we create a link instead? or expose the search by in Make.defs:
https://github.com/apache/incubator-nuttx-apps/blob/master/crypto/libtomcrypt/Make.defs#L26-L27

Reply:
I already tried that but it will not work since several headers have circular dependency

Reply:
tested on nnabla and works fine